src/appleseedmaya/shadingnoderegistry.cpp

```diff
@@ -87,8 +87,6 @@ namespace
 
         if (p.isStruct)
             std::cout << "    StructName = " << p.structName << "\n";
-
-        std::cout << "    ValidDefault = " << p.validDefault << "\n";
```

Comment:
Why do you want to remove this?

Reply:
Why do we output anything to cout?

Reply:
This function is a debugging helper that we enable when needed and disable after.
It is not used in normal plugin operation.